src/Dropdown.js

```diff
@@ -1,60 +1,132 @@
-              <StyledOcticon icon={TriangleDown} size={14} ml={title ? 2 : 0} />
-            </Button>
-            <BorderBox
-              bg="white"
-              border={1}
-              borderColor="gray.2"
-              borderRadius={1}
-              boxShadow="small"
-              mt={1}
-              px={3}
-              py={2}
-              minWidth={minWidth}
-              css={{position: 'absolute', zIndex: 99999}}
-            >
-              {children}
-              <Caret location="top-left" />
-            </BorderBox>
-          </React.Fragment>
-        )}
-      </Flex>
-    </div>
+    <Button is="summary" aria-haspopup="true" {...rest}>
+      {children}
+      <DropdownCaret />
+    </Button>
   )
 }
 
-const Dropdown = styled(DropdownBase)(COMMON)
+const DropdownMenu = styled.ul`
+  background-clip: padding-box;
+  background-color: ${get('colors.white')};
+  border: 1px solid rgba(27, 31, 35, 0.15);
+  border-radius: ${get('space.1')}px;
+  box-shadow: 0 3px 12px rgba(27, 31, 35, 0.15);
+  left: 0;
+  list-style: none;
+  margin-top: 2px;
+  padding: 5px 0 5px 0 !important; //TODO: fix this override on our markdown styles
+  position: absolute;
+  top: 100%;
+  width: 160px;
+  z-index: 100;
+
+  &::before {
+    position: absolute;
+    display: inline-block;
+    content: '';
+  }
+
+  &::after {
+    position: absolute;
+    display: inline-block;
+    content: '';
+  }
+
+  &::before {
+    border: ${get('space.2')}px solid transparent;
+    border-bottom-color: ${get('colors.blackfade15')};
+  }
+
+  &::after {
+    border: 7px solid transparent;
+    border-bottom-color: ${get('colors.white')};
+  }
+
+  // stylelint-disable-next-line selector-max-type
+  > ul {
+    list-style: none;
+  }
+  ${props => (props.direction ? getDirectionStyles(props.theme, props.direction) : '')};
+  ${COMMON};
+`
+
+const DropdownItem = styled.li`
+  display: block;
+  padding: ${get('space.1')}px 10px ${get('space.1')}px 15px;
+  overflow: hidden;
+  color: $get('colors.gray.9');
+  text-overflow: ellipsis;
+  white-space: nowrap;
+
+  &:focus {
+    color: ${get('colors.white')};
+    text-decoration: none;
+    background-color: ${get('colors.blue.5')};
+  }
+
+  &:hover {
+    color: ${get('colors.white')}
+    text-decoration: none;
+    background-color: ${get('colors.blue.5')};
+    outline: none;
+  }
+  ${COMMON}
+`
 
-Dropdown.defaultProps = {
+Dropdown.Button = styled(DropdownButton)(COMMON)
+Dropdown.Menu = DropdownMenu
+Dropdown.Item = DropdownItem
+
+Dropdown.Menu.propTypes = {
+  direction: PropTypes.oneOf(['ne', 'e', 'se', 's', 'sw', 'w']),
+  ...COMMON.propTypes
+}
+
+Dropdown.Menu.defaultProps = {
+  direction: 'sw',
   theme
 }
 
+Dropdown.defaultProps = {theme}
 Dropdown.propTypes = {
   children: PropTypes.node,
-  scheme: Button.propTypes.scheme,
-  title: PropTypes.string,
+  ...COMMON.propTypes
+}
+
+Dropdown.Button.defaultProps = {theme}
+Dropdown.Button.propTypes = {
+  ...COMMON.propTypes
```

Comment:
Does this go against the docs above?

> Dropdown.Menu, Dropdown.Button, and Dropdown.Item do not get any system props.

Reply:
Oh thank you, good catch! I added that in last minute